CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ All notable changes to this project will be documented in this file.
 - Use `WorkerpoolAuthorization#getHash` instead of `AuthorizationService#getChallengeForWorker`. (#272)
 - Reorder static and final keywords. (#274)
 - Update methods visibility and remove redundant checks in `SecretSessionBaseService`. (#276)
+- Refactor `IexecEnvUtils` to use `dealParams` instead of deprecated `TaskDescription` fields. (#277)
```

Comment:
This sentence doesn't match what's being done in this PR. 
```suggestion
- Refactor `SecretSessionBaseService` to use `dealParams` instead of deprecated `TaskDescription` fields. (#277)
```